addons/l10n_ae/models/account_move_line.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields, api
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    tax_ids = fields.Many2many(string='VAT')
+    vat_amount = fields.Monetary(compute='_compute_vat_amount', string='VAT Amount')
```

Comment:
Seems overkill to have a field for this.  You could probably just calculate it on the fly in the qweb itself.

Reply:
Will that work on the tree view in the account.move form view? Because this will not only be viewed in the report but also in the frontend view.